app/Helpers/Payment/EventPayment.php

```diff
@@ -52,9 +52,14 @@ public function getPackagePrice(): int
         return $this->package === null ? 0 : $this->package->price;
     }
 
+    public function getDiscount(): float
+    {
+        return min($this->participant->incomeBasedDiscountFactor, $this->event->earlybirdDiscountFactor);
```

Comment:
Hoogste korting geldt dus... klinkt opzich logisch. Dus de vroegboekkorting is eigenlijk vooral voor ouders die normaal geen korting krijgen?

Reply:
Klopt. Al waren de letterlijke woorden van het bestuur "de vroeg boek korting telt niet als je inkomens afhankelijke korting heb" . Maar let's be honest, zal anderwijs ooit iemand die een income afhankelijke korting heeft die lager is dan de vroegboek korting die korting ontzeggen. Ik zal er nog wel melding van maken dat we het zo doen, maar dit lijkt me wel logischer.